pkg/util/procfs/procfs_linux.go

```diff
@@ -90,7 +90,7 @@ func PKill(name string, sig syscall.Signal) error {
 	return utilerrors.NewAggregate(errList)
 }
 
-// Find process(es) with a specified name (exact match)
+// Find process(es) with a specified name (regexp match)
```

Comment:
I feel like this is not addressing the issue. The argument is `name` in string, and the function calls `regexp` to make sure it's an exact match.